hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -1034,14 +1193,10 @@ def __init__(self, raw: int, op_vals: Dict[str, int]):
         self.wsr = op_vals['wsr']
         self.wrs = op_vals['wrs']
 
-    def execute(self, state: OTBNState) -> None:
-        if not state.wsrs.check_idx(self.wsr):
-            # Invalid WSR index. Stop with an illegal instruction error.
-            state.stop_at_end_of_cycle(ILLEGAL_INSN)
-            return
-
+    def execute(self, state: OTBNState, cont: object) -> object:
         val = state.wdrs.get_reg(self.wrs).read_unsigned()
         state.wsrs.write_at_idx(self.wsr, val)
+        return None
```

Comment:
What happened to the illegal instruction error on an invalid WSR index?

Reply:
It got deleted by accident! Oops...